app/src/main/java/org/coepi/android/domain/UnixTime.kt

```diff
@@ -28,4 +28,8 @@ class UnixTime private constructor(val value: Long) : Parcelable, Serializable {
 
     fun toDate() =
         Date(value * 1000)
+
+    fun toMonthAndDay() = DateFormatters.dotFormatter.formatMonthDay(toDate())
```

Comment:
This may be convenient, but formatting shouldn't be in domain classes. If you really need it, add it as an extension in the view models.